src/ray/core_worker/transport/direct_actor_transport.cc

```diff
@@ -166,7 +166,9 @@ void CoreWorkerDirectActorTaskSubmitter::ConnectActor(const ActorID &actor_id,
                 << queue->second.caller_starts_at << " to "
                 << queue->second.num_completed_tasks;
   queue->second.caller_starts_at = queue->second.num_completed_tasks;
+
   SendPendingTasks(actor_id);
+  ResendOutOfOrderTasks(actor_id);
```

Comment:
To make sure I'm understanding correctly: the sequence numbers of these may not be in order (e.g., we send some earlier tasks in the `ResendOutOfOrderTasks` than we do in `SendPendingTasks`, but that's ok because we're relying on the receiver-side ordering, right?

Reply:
Maybe we should call this `ResendIncompleteTasks` instead? Also, does it make sense to do this before `SendPendingTasks` (not sure if it makes a difference, but it seems more intuitive)?

Reply:
@edoakes your understanding is correct, i'll add a comment.
@stephanie-wang i'm confused. `ResendOutofOrderTasks` are resending tasks that were completed before again just so the receiver can keep track of all of seqnos. It really isn't `ResendIncompleteTasks`?